src/rust/bitbox02-rust/src/xpubcache.rs

```diff
@@ -17,12 +17,18 @@ use super::keystore;
 use crate::bip32;
 use alloc::vec::Vec;
 
+#[derive(Copy, Clone)]
+pub enum Compute {
```

Comment:
nit: Maybe it could be useful to add a comment here (or somewhere else) explaining that this is used as an additional bitflips check for sensitive operations

Reply:
Added comments.